CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+## How to Contribute to OpenMM Development
+
+We welcome anyone who wants to contribute to the project, whether by adding a feature,
+fixing a bug, or improving documentation.  The process is quite simple.
+
+First, it is always best to begin by opening an issue on Github that describes the change you
+want to make.  This gives everyone a chance to discuss it before you put in a lot of work.
+For bug fixes, we will confirm that the behavior is actually a bug and that the proposed fix
+is correct.  For new features, we will decide whether the proposed feature is something we
+want and discuss possible designs for it.
+
+Once everyone is in agreement, the next step is to create a pull request with the code changes.
```

Comment:
You can link to https://help.github.com/en/articles/about-pull-requests

Reply:
Good idea, I'll add that.